rskj-core/src/test/java/co/rsk/jsontestsuite/LocalBlockTest.java

```diff
@@ -35,10 +38,11 @@
 @FixMethodOrder(MethodSorters.NAME_ASCENDING)
 public class LocalBlockTest {
     private ClassLoader loader = LocalBlockTest.class.getClassLoader();
+    private static RskSystemProperties config = new RskSystemProperties();
```

Comment:
Probably we should remove this _static_ fields in some upcoming refactor

Reply:
The class uses the same configuration for all the tests. We can remove the static part but we would need to disable remasc before each test.

Reply:
Or maybe we can have a `@Before` method where we always create a new instance of the `RskSystemProperties`